### ISSUE TITLE
[1.12.X]Suggestion see what kind of mob spawner it is like waila

### ISSUE THREAD

**Issue by mcdodge34** [McJty/TheOneProbe#203]
Just a suggestion here, I just noticed that I can't tell which mob spawner I pickup, it just says monster spawner, while waila was able to tell me what mob it was, I know, the icon inside should tell me already, but when we work with a crap ton of spawners, its sometimes a pain to sort them out.

**Comment by McJty** [comment]
Do you mean when the spawner is still in the world or when it is picked up in inventory?

**Comment by mcdodge34** [comment]
Both, I want to see what type of mob will spawn out of it, I know the icon of the mob inside the spawner should tell me, but when I work with a crap ton of them, its just something I was very addicted with waila, I'm picking up the spawners with Chest Transporter mod for now, planning a mega mob spawner with some moving tracks where I can choose what mob I want to spawn, with lights that prevent spawning in a room where I have skeleton spawners if I want zombie, only zombie will spawn and such.

**Comment by McJty** [comment]
Ok, I can't do anything about the inventory item as that's something that The One Probe doesn't do (there are other mods for that) but I can do something about the spawner when it is in the world

**Comment by mcdodge34** [comment]
Yup, that's more what I was referring to, I think the mob spawner type in inventory does come up with the Chest Transporter, but in the world, its where it would be useful to know.

**Comment by TheSnowyChickens** [comment]
![2018-07-02_19 05 58](https://user-images.githubusercontent.com/29357962/42176942-00dae470-7e2b-11e8-990e-2fe47b7849ab.png)

TOP shows an error.

**Comment by mcdodge34** [comment]
Yup, still a problem, I'm unsure why the magic of Mcjty didn't apply to this long overdue problem... Just an annoyance, but I'm sure its easy to solve.

**Comment by McJty** [comment]
I simply never get around to it. I'll try to find some time to look into this

**Comment by mcdodge34** [comment]
Take the time needed, no pressure, I'm enjoying another game for awhile called 7 days to die modded with War Of The Walkers mod for now, I'm taking a Minecraft and Modded Minecraft break, still keep myself in touch with new stuff around and may get a Vanilla game started once the official 1.13 is released, but I'm in no hurry for now.

**Comment by EvilEtho** [comment]
This is still an issue. Do we know when this might be fixed? :)
![spawner](https://user-images.githubusercontent.com/6267767/148288047-57104b12-b75b-4469-9ba1-c22250c05c4b.JPG)

**Comment by McJty** [comment]
I no longer develop on 1.12 but if this bug also occurs in 1.16.5 I can fix it there. However I've never seen it. Not even on 1.12

**Comment by EvilEtho** [comment]
Oh my bad this is 1.18 I might have clicked in to the wrong version for bug reporting

**Comment by McJty** [comment]
Ok that's something else. I'll check it
